include/jwt-cpp/jwt.h

```diff
@@ -2873,11 +2875,243 @@ namespace jwt {
 		}
 	};
 
+	template<typename json_traits>
+	class jwk {
+		using basic_claim_t = basic_claim<json_traits>;
+		details::map_of_claims<json_traits> jwk_claims;
```

Comment:
there are no setters currently

```suggestion
		const details::map_of_claims<json_traits> jwk_claims;
```